src/azure-cli/azure/cli/command_modules/iot/_help.py

```diff
@@ -764,6 +764,27 @@
     text: |
         az iot hub update --name MyIotHub --set properties.allocationPolicy="GeoLatency"
     crafted: true
+  - name: Update units of an IoT Hub to 2
+    text: >
+        az iot hub update -n MyIotHub --unit 2
+  - name: Update pricing tier for an IoT Hub as S2
+    text: >
+        az iot hub update -n MyIotHub --sku S2
+  - name: Update the retention days of an IoT Hub device-to-cloud events as 3 days
+    text: >
+        az iot hub update -n MyIotHub --retention-day 3
+  - name: Update the IoT Hub cloud-to-device messaging settings
+    text: >
+        az iot hub update --name MyIotHub --c2d-max-delivery-count 30 --c2d-ttl 5
+  - name: Update the IoT Hub feedback queue settings
+    text: >
+        az iot hub update --name MyIoTHub --feedback-max-delivery-count 20 --feedback-lock-duration 100 --feedback-ttl 4
+  - name: Update the IoT Hub file upload settings
+    text: >
+        az iot hub update -n MyIoTHub --fileupload-sas-ttl 5 --fileupload-storage-auth-type identityBased
+  - name: Update the IoT Hub file upload notification settings
+    text: >
+        az iot hub update -n MyIoTHub --fnd 50 --fnt 48 --fn true
```

Comment:
Do you need `--fn true` or will `--fn` work in the turning on case. It looks more concise without the bool. I'd expect 'false' to be passed in if you need to turn it off.

Reply:
Correct